ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedgerWithMutableCache.scala

```diff
@@ -42,15 +49,26 @@ private[index] object ReadOnlySqlLedgerWithMutableCache {
           ledgerEndSequentialId,
         ).acquire()
         generalDispatcher <- dispatcherOwner(ledgerEndOffset).acquire()
-        contractStore <- contractStoreOwner(generalDispatcher, contractStateEventsDispatcher)
-        ledger <- ledgerOwner(contractStateEventsDispatcher, generalDispatcher, contractStore)
+        dispatcherLagger <- Resource.successful(
```

Comment:
nit: a lagger is someone that lags behind, drags his/her feet. The name you gave to the class, is better delivering the true purpose

Reply:
I'll rename 👍